cache-config/testing/docker/ort_test/Dockerfile

```diff
@@ -28,15 +28,17 @@
 # The recommended minimum size for each block devices is 1G.
 # For example, `sudo modprobe brd rd_size=1048576 rd_nr=2`
 
-FROM centos:7
+FROM centos:${RHEL_VERSION:-8}
```

Comment:
Because there is no [`ARG`](https://docs.docker.com/engine/reference/builder/#arg) instruction for `RHEL_VERSION`, there is no way to set `RHEL_VERSION` to `7` at build time.

Reply:
updated, added RHEL_VERSION to the docker-compose.yml ort build stage.

Reply:
@zrhoffman fixed